contents/blog/changing-to-self-serve.md

```diff
@@ -0,0 +1,74 @@
+---
+date: 2022-05-19
+title: Next Up - Nailing Self Serve
```

Comment:
Trying to think of title that doesn't involve the word 'pivot'.

Probably something like:

- Why 'Nailing Self Serve' is PostHog's new focus
- We're focused on 'Nailing Self Serve'. Here's why

Reply:
I think these suggestions both sound like the company pivoted, whereas we just clarified what we need to tackle next.

I'm not really writing this with HN in mind,  it's more a story for fans - I prefer mine at the moment but I'm open to more suggestions if you have any (I don't)